public/app/components/fileTree/fileTree.js

```diff
@@ -0,0 +1,84 @@
+angular.module('kibibitCodeEditor')
+
+.directive('kbFileTree', function() {
+  return {
+    scope: {},
+    bindToController: {
+      path: '=kbFileTreePath',
+      kbFileTreeSelection: '=',
```

Comment:
use `kbXXYY` as an html attribute, but `XXYY` as a variable.

so, these two should be:
```
selection: '=kbFileTreeSelection',
options: '=kbFileTreeOptions'
```
(and in the controller use `vm.selection` & `vm.options`)